packages/x-components/src/x-modules/url/components/__tests__/url-handler.spec.ts

```diff
@@ -16,23 +16,27 @@ import { UrlHandler } from '../index';
 function renderUrlHandler({ template = `<UrlHandler />` }: UrlHandlerOptions = {}): UrlHandlerAPI {
   const [, localVue] = installNewXPlugin({ initialXModules: [urlXModule] });
 
-  const wrapperTemplate = mount(
-    {
-      template,
-      components: {
-        UrlHandler
-      }
-    },
-    {
-      localVue
-    }
-  );
+  const wrapperTemplate = mount({ template, components: { UrlHandler } }, { localVue });
   const wrapper = wrapperTemplate.findComponent(UrlHandler);
-  const $x = wrapperTemplate.vm.$x;
+
+  function getUrlFunction(event: string) {
```

Comment:
If you're dispatching an event I'l rename the method, right now confused more than anything else.

Reply:
done